examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -469,14 +471,15 @@ class MapboxNavigationActivity : AppCompatActivity() {
                 .applyDefaultNavigationOptions()
                 .applyLanguageAndVoiceUnitOptions(this)
                 .coordinatesList(listOf(origin, destination))
+                .alternatives(true)
```

Comment:
I didn't want to commit this. Will revert later 🙂

Reply:
BTW, what could a reason to not keep it in the internal example?

Reply:
I think it might affect example like it. If you want to extend alternatives refresh please address it to an appropriate example

Reply:
got rid of alternatives and and route refresh option in the example